Cabal/Distribution/PackageDescription/Configuration.hs

```diff
@@ -63,7 +63,8 @@ import Distribution.Types.CondTree
 import Distribution.Types.Condition
 import Distribution.Types.DependencyMap
 
-import qualified Data.Map as Map
+import qualified Data.Map.Strict as Map.Strict
```

Comment:
this module doesn't exist in `containers-0.4`. We have `Cabal.Distribution.Compat.Map` to deal with that.

Reply:
Actually, it looks like `Data.Map.Lazy` also doesn't exist yet there is not a compatibility wrapper for this. I suppose one should be introduced.

Reply:
Yes. I think there we need the lazy variant too.